scripts/xsslint/xsslint/main.py

```diff
@@ -18,6 +19,23 @@ def _load_config_module(module_path):
     return importlib.import_module(module_path)
 
 
+def _build_ruleset(template_linters):
+    """
+    Combines the RuleSets from the provided template_linters into a single, aggregate RuleSet.
+
+    Arguments:
+        template_linters: A list of linting objects.
+
+    Returns:
+        The combined RuleSet.
+    """
+    return reduce(
```

Comment:
Python question: Is this basically doing the equivalent of the following code, but using a loop?:
`self.ruleset = self.ruleset + self.javascript_linter.ruleset + self.python_linter.ruleset`

This could possibly be a static factory method on RuleSet that takes a list of objects with a ruleset method.  The above code would then be:
```
self.ruleset = RuleSet.build_ruleset([self, self.javascript_linter, self.python_linter]);
```
I'm just thinking out loud.  No changes are required.

Reply:
Yes, it loops through the list of linters and reduces them to a single value (the combined RuleSet).